doc/source/how-to/dns-configuration.rst

```diff
@@ -0,0 +1,93 @@
+DNS configuration
+=================
+
+As explained in :ref:`Documenting`, PyAnsys projects publish their documentation
+online under the following canonical name (CNAME) convention:
+
+``https://<product>.docs.pyansys.com``
+
+In order to request a CNAME for the ``pyansys.com`` domain, please contact
+`Maxime Rey`_, `Roberto Pastor Muela`_ or `Alex Kaszynski`_. Any of these members
+can handle the creation of the requested PyAnsys subdomain.
+
+Once the CNAME is created, repository administrators can configure their published
+documentation in GitHub pages to be exposed through it. In order to configure the CNAME
+for your documentation, please refer to GitHub's online documentation:
+`Managing a custom domain for your GitHub Pages site`_.
+
+PyAnsys verified domains
+------------------------
+
+In the `PyAnsys organization`_, the following domains have been verified:
+
+* ``pyansys.com``
+* ``docs.pyansys.com``
+
+.. warning::
+
+    Only CNAME requests with **one** subdomain before the previous verified
+    domains are allowed. The reasons behind this measure are explained in
+    :ref:`DNS protection measures`.
+
+DNS protection measures
+-----------------------
+
+The rationale behind choosing the previous CNAME convention is due to cybersecurity reasons.
+GitHub allows to verify domains for users and organizations. In order to do so, GitHub's online
+documentation can be found here: `Verifying a domain for your organization site`_.
+
+Once an organization has a verified domain, **this prevents external users to the organization from
+taking over existing direct subdomains**. However, deeper subdomains are no longer verified by
+GitHub.
+
+This is better explained with the following examples:
+
+Case scenario - **protected** subdomain
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Consider that the domain ``pyansys.com`` has been verified for the `PyAnsys organization`_.
+#. A CNAME as follows is requested: ``subdomain.pyansys.com``
+
+The CNAME requested can only be used by projects/repositories inside the `PyAnsys organization`_.
+Any attempt by external users/organizations of taking over the previous CNAME is taken
+care by GitHub and rejected.
```

Comment:
WHat about rewriting this as to avoid using "take" twice? 

```suggestion
Any attempt by external users/organizations of taking over the previous CNAME is identifiead and rejected by GitHub.
```